docs/bootstrap.json

```diff
@@ -0,0 +1,20 @@
+{
+  "ID": "bootstrap",
+  "Properties": {
+    "Resources": {
+      "resource_a": {
+        "Plugin": "resource-file",
+        "Type": "a",
+        "Properties": {
+          "Note": "Depends on {{.Resources.resource_b.ID}}"
+        }
+      },
+      "resource_b": {
+        "Plugin": "resource-file",
+        "Type": "b",
```

Comment:
We may want to simplify this to follow what Microsoft does in Azure for their resource type:
`"type": "Microsoft.Storage/storageAccounts"`

The above would become:
```
"resource_b": {
  "Plugin": "resource-file/b",
  "Properties": {
  }
}
```

This would also us to re-use the instance type plugin